src/instances/RaidInstance.ts

```diff
@@ -421,19 +432,19 @@ export class RaidInstance {
         }
 
         this._earlyLocPointCost = costForEarlyLoc;
-        this._locationToProgress = locationToProgress;
+        this._locationToProgress = (this._vcless) ? true : locationToProgress;
 
-        if (vcLimit === -2) {
+        if (raidLimit === -2) {
             if (section.otherMajorConfig.afkCheckProperties.vcLimit !== -1)
-                vcLimit = section.otherMajorConfig.afkCheckProperties.vcLimit;
-            else vcLimit = 60;
+                raidLimit = section.otherMajorConfig.afkCheckProperties.vcLimit;
+            else raidLimit = 45;
```

Comment:
Why 45?

Reply:
O3 is the only dungeon that can even have more than 45 people. Why would the default exceed that?  Besides, this is overridden in configs most of the time anyways

Reply:
Makes sense. I forgot about the dungeon cap across all dungeons